pkg/network/usm/utils/debugger.go

```diff
@@ -162,6 +167,19 @@ func (d *tlsDebugger) GetBlockedPathIDs(programType string) []PathIdentifier {
 	return nil
 }
 
+// ClearBlocked clears the list of blocked paths for all registries.
+func (d *tlsDebugger) ClearBlocked() {
+	d.mux.Lock()
+	defer d.mux.Unlock()
+
+	for _, registry := range d.registries {
+		registry.m.Lock()
+		defer registry.m.Unlock()
+
+		registry.blocklistByID.Purge()
+	}
```

Comment:
I think that's antipattern to use defer within a loop, although it was fixed in go1.22, I do recommend to not use this pattern
either manually call unlock at the end of the iteration, or wrap with a function

Reply:
Fixed.